ingest/user_de.py

```diff
@@ -164,21 +212,70 @@ def generate_individual_files(col, genes, rest, groups, clean_val, qual):
         arr = np.array(file_d[i])
         t_arr = arr.transpose()
         inner_df = pd.DataFrame(data = t_arr, columns = qual)
+
+        #inner_df.to_csv(f'{cluster_name}--{clean_annotation}--{clean_group}--{annot_scope}--{method}.tsv'.format(cluster_name, clean_annotation, clean_group, annot_scope,i), sep ='\t')
         inner_df.to_csv(f"ingest/{i}.tsv".format(i), sep ='\t')
+
         final_files_to_find.append("ingest/{}.tsv".format(i))
+        print(final_files_to_find)
+
     return final_files_to_find
 
 # final result: individual files for each comparison
 
-if __name__ == '__main__':
-    data = pd.read_csv(file_path)
-    long_format = convert_wide_to_long(data)
-    wide_format = convert_long_to_wide(long_format)
+def generate_manifest(clean_val, clean_val_p, qual):
+    """
+    create manifest file of each comparison in the initial data
+    if the comparison is with rest, rest is omitted and just the type is written
+    """
+    file_names_one_vs_rest = []
+    for i in clean_val:
+        cleaned_list = [ x for x in i if x != "rest" and x not in qual]
+        file_name_one = '_'.join(cleaned_list)
```

Comment:
Blocker: please see Jon's note in his review about [output filename format](https://github.com/broadinstitute/scp-ingest-pipeline/pull/309#discussion_r1220107692).  

### Delimiter
The delimiter for each "segment" (like "type_0") of the file name must be a string that does not appear within any segment.  

Here, the current delimiter is "_".  Underscores can also appear internal in a filename segment, e.g. "type_3".  So we need to use a different string as a delimiter.

Ingest Pipeline conventionally uses two hyphens for such delimiters.  Let's apply that here.

```
        clean_comparison = '_'.join([ x for x in i if x != "rest" and x not in qual])
```

### Context segments
Above in this file, you have placeholders.  Here they are, with more realistic values:
```
cluster_name = "All_Cells_UMAP"
clean_annotation = "General_Celltype"
annot_scope = "study"
method = "wilcoxon"
```

As Jon notes in that link, the output filenames need to contain these values (here is replace `clean_group` from Jon with a clearer variable name `clean_comparison`).  This would be something like:

file_name_one = f"{cluster_name}--{clean_annotation}--{clean_comparisons}--{annot_scope}--{method}"

More concretely, we need file names like:

* Pairwise: `All_Cells_UMAP--General_Celltype--type_2--type_10--study--wilcoxon.tsv`

* One-vs-rest: `All_Cells_UMAP--General_Celltype--type_1--study--wilcoxon`
  * ^ Note how "rest" is omitted in `clean_comparisons`.

Reply:
file names changed, rest omitted from file name